src/frontEnd/volunteering/findVolunteeringOpportunities.ts

```diff
@@ -1,14 +1,27 @@
-import { volunteeringOpportunities } from "./volunteeringOpportunities";
-import { anyOf } from "../../utils/predicates/anyOf";
+import { defaultOrganizations } from "../organizations/data/defaultOrganizations";
 
 export const findVolunteeringOpportunities = (orgOrJobTitle) => {
+
     const inputValue = (orgOrJobTitle || "").trim().toLowerCase();
-    return inputValue.length === 0
-        ? []
-        : volunteeringOpportunities.filter(
-            anyOf(
-                option => option.title.toLocaleLowerCase().includes(inputValue),
-                option => option.jobTitle.toLocaleLowerCase().includes(inputValue)
-            )
-        );
+
+    const filterData = () => {
+        const results = defaultOrganizations.map(element => {
```

Comment:
Here I think, given the list of org/volunteering opportunity pairs above, your filter is simpler, it returns pairs where the sought string appears in any of the properties.

After that, u can just map the results to whatever structure u need; or u can map first and then search